test/sharness/t0020-service-startup-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# MIT LICENSED
+#
+test_description="Test service startup and init functionality"
+
+. lib/test-lib.sh
+
+test_expect_success "ipfs is installed on this machine" '
+   ipfs help >ipfs_help.txt &&
+   egrep -i "^Usage" ipfs_help.txt >/dev/null  
+'
+
+test_expect_success "launch ipfs daemon" '
+   mkdir -p ../.test_ipfs &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs init'"'"' &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs daemon & echo $! >../dPID.txt'"'"' &&
```

Comment:
This evals are very esoteric. Can you try making a separate regular bash function to wrap ipfs launch ipfs and see if the `echo $!` works without evals and stuff there?

Also, store the PIDs in a variable. No need to write files for that.